pkg/kademlia/kademlia.go

```diff
@@ -245,6 +246,28 @@ func (k *Kademlia) FetchInfo(ctx context.Context, node pb.Node) (*pb.InfoRespons
 	return info, nil
 }
 
+// FetchGraph connects to a node address and returns its routing table as a graph
+func (k *Kademlia) FetchGraph(ctx context.Context, node pb.Node) (*pb.GraphResponse, error) {
```

Comment:
Yeah, I think since we don't want to make it public, we don't have to add it as a public thing to kademlia. This means we'll avoid accumulating diagnostic methods on kademlia.